src/main/java/seedu/exercise/logic/parser/Prefix.java

```diff
@@ -1,8 +1,8 @@
-package seedu.address.logic.parser;
+package seedu.exercise.logic.parser;
 
 /**
  * A prefix that marks the beginning of an argument in an arguments string.
- * E.g. 't/' in 'add James t/ friend'.
+ * E.g. 't/' in 'add Run t/ Leg'.
```

Comment:
May want to rewrite "t/" to "m/" to adhere to our prefixes.

Reply:
done